src/playbacks/hls/hls.js

```diff
@@ -119,6 +133,22 @@ export default class HLS extends HTML5VideoPlayback {
     this.trigger(Events.PLAYBACK_SETTINGSUPDATE)
   }
 
+  onError(evt, data) {
+    if (data && data.fatal) {
+      switch (data.type) {
+        case Hls.ErrorTypes.NETWORK_ERROR:
+          hls.startLoad()
```

Comment:
Should there maybe be some kind of delay before trying to load again? Don't want to start a dos attack on the server if it is returning an error :P

And maybe a limit on the number of attempts?